improve coverage on price protection and slippage (#31)
latent-to/developer-docs: docs/glossary.md

$$R_j = \sum_{i \in \text{validators}} S_i \cdot \overline{W_{ij}}$$

Where:

- $S_i$ is the stake of validator $i$
- $\overline{W_{ij}}$ is the consensus-clipped weight from validator $i$ to neuron $j$
- The sum is taken over all validators in the subnet

**Source**:

- [`bittensor/bittensor/core/metagraph.py:325-331`](https://github.com/opentensor/bittensor/blob/main/bittensor/core/metagraph.py#L325-331)
- [`subtensor/pallets/subtensor/src/epoch/run_epoch.rs:605`](https://github.com/opentensor/subtensor/blob/main/pallets/subtensor/src/epoch/run_epoch.rs#L605)

### Recycling, burning, and locking

> "Recycling TAO" means that this TAO is put back into the Bittensor emissions system. Instead of minting new TAO this recycled TAO that is in the recycle bin will be used again in the new emissions.

Reviewer: ```suggestion
"Recycling TAO" means that this TAO is put back into the emission pool. Instead of minting new TAO this recycled TAO will be distributed again later.
```

This happens in two cases:

> - When you register either as a subnet validator or a subnet miner and get a `UID` in return, the registration cost TAO you pay is recycled.

Reviewer: ```suggestion
- When you register either as a subnet validator or a subnet miner and get a `UID` in return, the resulting ALPHA from the registration's swap is recycled.
```

> - Emissions are recycled for those subnets that have registration turned off or paused.

Reviewer: ```suggestion
- Emissions are recycled for those subnets that have their registration turned off.
```

> When TAO is burned it is permanently removed from circulation, reducing total supply.

Reviewer: It's removed from circulation, but doesn't reduce the total supply or issuance. Also TAO can't be burned directly you can only burn ALPHA. So the only way to "burn" TAO would be swapping TAO for ALPHA and burn the resulting ALPHA.

**See also:** [Subnet Miners](./miners/), [Subnet Validators](./validators/), [Working with Subnets](./subnets/working-with-subnets.md)

## S

### SS58 Encoded

A compact representation of public keys corresponding to the wallet's coldkey and hotkey, used as wallet addresses for secure TAO transfers.

**See also:** [Working with Keys](./working-with-keys.md), [Wallets](./getting-started/wallets.md)

### Slippage

> In the context of an automated market maker (AMM), slippage is the impact on the token yield of a trade of the change in price due to the trade transaction itself.

Reviewer: ```suggestion
In the context of an automated market maker (AMM), slippage is the impact on the token acquired in a trade due to the change in price from the trade transaction itself.
```

Reviewer: As mentioned in the `price-protection.md` yield is used for rewards over time.

For a validator with alpha stake $\alpha$ and TAO stake $\tau$, the stake weight $W$ is calculated as:

$$
W = {\alpha + \tau \ \times w_{\tau}}
$$

Where $w_{\tau}$ is the global TAO weight parameter (currently 0.18)

A validator's relative influence in a subnet is calculated as:

$$
\text{Relative Stake Weight} = \frac{\text{Stake Weight}_i}{\sum_{v \in \text{validators}} \text{Stake Weight}_v}
$$

**Consensus Power:**

- **Weight Setting**: Higher stake weight means more influence when setting weights
- **Validator Permits**: Stake weight determines eligibility for validator permits
- **Bond Formation**: Stake weight influences bond calculations and retention

**Validator Emissions:**

- **Relative Distribution**: Higher stake weight -> higher emission share

**Code References:**

- **Yuma Consensus**: [`subtensor/pallets/subtensor/src/epoch/run_epoch.rs:530`](https://github.com/opentensor/subtensor/blob/main/pallets/subtensor/src/epoch/run_epoch.rs#L530)
- **Validator dividend distribution**: [`subtensor/pallets/subtensor/src/coinbase/run_coinbase.rs:165`](https://github.com/opentensor/subtensor/blob/main/pallets/subtensor/src/coinbase/run_coinbase.rs#L165)

### Staking

> The process of attaching TAO to a validator hotkey, i.e., locking TAO to a subnet validator's hotkey to increase their total stake and increase their consensus power and share of subnet emissions.

Reviewer: ```suggestion
The process of attaching TAO to a validator hotkey, i.e., locking TAO to a subnet validator's hotkey to increase their total stake and increase their consensus power and share of dividends.
```

Reviewer: You aren't increasing their share of subnet emissions (that is 41% for validators), you are increasing their share of the dividends (% of the 41%). Also staking technically can be done to any hotkey like a miner hotkey.
